Core/OfficeDevPnP.Core/Framework/Provisioning/Providers/Xml/PnPObjectsMapper.cs

```diff
@@ -194,6 +194,7 @@ public static void MapProperties(Object source, Object destination, Dictionary<S
                                     {
                                         sourceValue = null;
                                     }
+                                    spSpecified.SetValue(destination, isSpecified);
```

Comment:
I think this one is wrong.

Reply:
I had to add that one to make it work over here...